requirements.txt

```diff
@@ -10,5 +10,3 @@ zimscraperlib>=1.3.0,<1.4
 kiwixstorage>=0.3,<1.0
 pif>=0.8.2,<0.9
 xxhash>=2.0.0,<2.1
-# youtube-dl should be updated as frequently as possible
```

Comment:
Although it's not a direct dependency anymore, we should keep it so a `pip install -U -r requirements.txt` updates it even if there's no new scraperlib version.